src/CalcViewModel/DateCalculatorViewModel.cpp

```diff
@@ -43,11 +43,7 @@ DateCalculatorViewModel::DateCalculatorViewModel() :
     m_StrDateDiffResultAutomationName(L""),
     m_StrDateDiffResultInDays(L""),
     m_StrDateResult(L""),
-    m_StrDateResultAutomationName(L""),
-    m_fromDate({ 0 }),
```

Comment:
What's your argument for removing the explicit initialization here?

Reply:
These 4 objects are initialized in the constructor lines 61-65, these 4 lines weren't necessary